ethereum/networks/src/main/java/tech/pegasys/teku/networks/Eth2NetworkConfiguration.java

```diff
@@ -580,11 +577,6 @@ public Builder epochsStoreBlobs(final String epochsStoreBlobs) {
       return this;
     }
 
-    public Builder kzgNoop(final boolean kzgNoop) {
-      this.kzgNoop = Optional.of(kzgNoop);
-      return this;
-    }
-
     public Builder applyNetworkDefaults(final String networkName) {
```

Comment:
Just noticed this.
Could we put "mainnet-trusted-setup.txt" and "minimal-trusted-setup.txt" in static constants also?

Reply:
I am following the pattern how we specify `mainnet-genesis.ssz` for loading files from classpath. Not sure we want to change this for the trusted setup.

Reply:
I mean just
`final private static String MINIMAL_TRUSTED = "minimal-trusted-setup.txt";`
You are probably didn't get me

Reply:
yeah that makes sense, done!